### ISSUE TITLE
No Python 3.6 package for argcomplete

### ISSUE THREAD

**Issue by niklasholm** [ContinuumIO/anaconda-issues#7517]
Attempted to install the tab completion package for my (mini)conda install:

```
$ conda install argcomplete
Fetching package metadata ...........
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - argcomplete -> argparse -> python 2.6*
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.

$ conda --version
conda 4.3.30

$ uname -srm
Linux 3.13.0-135-generic x86_64
```

Looking at https://anaconda.org/anaconda/argcomplete/files, it seems no package for Python 3.6 is provided.

**Comment by paicheng** [comment]
I do not know if this works:
1. create an environment `py35` with packages `python 3.5` and `argcomplete` 
`conda create -n py35 python=3.5 argcomplete`

2. activate `py35`
`activate py35`

**Comment by niklasholm** [comment]
Then I would need to have that environment activated all the time which kind of defeats the point.

**Comment by jjhelmus** [comment]
argcomplete 1.9.3 packages are now available for Python 2.7, 3.5 and 3.6.
